Testscripts/Linux/utils.sh

```diff
@@ -2355,18 +2355,19 @@ function install_sshpass () {
 	which sshpass
 	if [ $? -ne 0 ]; then
 		echo "sshpass not installed\n Installing now..."
-		install_package sshpass
-		which sshpass
+		check_package "sshpass"
 		if [ $? -ne 0 ]; then
 			echo "sshpass not installed\n Build it from source code now..."
 			package_name="sshpass-1.06"
 			source_url="https://sourceforge.net/projects/sshpass/files/sshpass/1.06/$package_name.tar.gz"
 			wget $source_url
 			tar -xvf "$package_name.tar.gz"
 			cd $package_name
-			install_package "gcc make"
+			install_package "gcc make wget"
```

Comment:
this has to be moved right after the echo above - line 2360.
you already have a wget command above this install_package, so the order is off.

and while at it, please remove the verbose flag from tar extract.